entity-framework/core/miscellaneous/dbcontextpool.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Connection pooling
+description: Connection pooling in Entity Framework Core
+author: rick-anderson
+ms.author: riande
+ms.date: 9/19/2020
+uid: core/miscellaneous/dbcontextpool
+---
+# Connection pooling
+
+`AddDbContextPool` enables `DbContext` pooling. Context pooling can increase throughput in high-scale scenarios such as web servers by reusing `DbContext` instances, rather than creating new instances for each request.
```

Comment:
We generally avoid backticking common type names to make the text flow better. For DbContext specifically, we usually just talk about contexts unless it's specifically ambiguous (except possibly the first mention in the document) - does that makes sense? If so, can you change that throughout the document?

```suggestion
`AddDbContextPool` enables pooling of DbContext instances. Context pooling can increase throughput in high-scale scenarios such as web servers by reusing `DbContext` instances, rather than creating new instances for each request.
```